tracks/bash/exercises/grains/mentoring.md

```diff
@@ -13,13 +13,11 @@ function main() {
 
   if [[ "$square" == "total" ]]; then
     echo "2^64 - 1" | bc
-    exit 0
-  elif [[ "$square" -lt 1 || "$square" -gt 64 ]]; then
+  elif (( square < 1 || square > 64 )); then
     echo "Error: invalid input"
-    exit 1
+    return 1
   else
     echo "2^($square - 1)" | bc
```

Comment:
Want to incorporate herestrings while you are at it?

Reply:
@IsaacG Sure, done